<?xml version="1.0"?>

<launch>
	<include file="$(find final_assignment)/launch/simulation_gmapping.launch"></include>
	<include file="$(find final_assignment)/launch/move_base.launch"></include>

	<arg name="target_x" default= "0"/>
    <arg name="target_y" default= "0"/>
    <arg name="user_command" default= "0"/>
    <arg name="target_reached" default = "True" />
    
    <param name="target_x" value="$(arg target_x)"/>
    <param name="target_y" value="$(arg target_y)"/>
    <param name="user_command" value="$(arg user_command)"/>
    <param name="target_reached" value="$(arg target_reached)"/>
    
    <node pkg="final_assignment" type="user_target_server.py" name="user_target_server" output = "screen"/>
    <node pkg="final_assignment" type="user_command_server.py" name="user_command_server" output = "screen"/>
    <node pkg="final_assignment" type="wall_follow_service_m.py" name="wall_follower"  />
    <node pkg="final_assignment" type="go_to_point_service_m.py" name="go_to_point" />
    <node pkg="final_assignment" type="bug_m.py" name="bug" output="screen"/> 
    <node pkg="final_assignment" type="random_target_server.py" name="random_target_server" output = "screen"/>
    <node pkg="final_assignment" type="robot_controller.py" name="robot_controller" output = "screen"/> -->
    <!--<node pkg="final_assignment" type="prova.py" name="prova" output = "screen"/> -->
    
</launch>
